### ISSUE TITLE
[4.0.0-rc.1] Split Pane: scrolling on the right side disables tap events on the left side

### ISSUE THREAD

**Issue by elwinvaneede** [ionic-team/ionic#17096]
# Bug Report

**Ionic version:**
[x] **4.0.0-rc.1**

**Current behavior:**

When you use the Split Pane Layout (ion-split-pane), and you scroll the right pane, you can't tap buttons/menu items in the left pane while scrolling (and vice versa). This mainly manifests when running on a tablet device (tested with iOS) and scrolling through long content. You have to wait for the scroll animation to end before allowed to tap a button in the left pane.

**Expected behavior:**

I understand why the tap events in the right pane are disabled while scrolling, but I don't expect it to be the case in the left pane. The iOS Settings app (on an iPad) doesn't exhibit this behaviour for example.

**Steps to reproduce:**

1. Use ion-split-pane with a menu on the left page and a lot of overflowing content in the right page (see related code).
2. Create an iOS Cordova build
3. Run on iPad
4. Scroll the right pane and while it is still scrolling try to tap on a menu item in the left pane (only responds when the scrolling has stopped)

**Related code:**

Sample application via GitHub: https://github.com/elwinvaneede/ionic-v4-split-pane-scroll-issue-example

**Other information:**

I don't know if this is specific to Ionic or more an Angular issue. Or if there is an easy fix, but I haven't found it.

**Ionic info:** 

```
Ionic:
   ionic (Ionic CLI)             : 4.7.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0-rc.1
   @angular-devkit/build-angular : 0.11.4
   @angular-devkit/schematics    : 7.1.4
   @angular/cli                  : 7.1.4
   @ionic/angular-toolkit        : 1.2.2

Cordova:
   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.1, (and 4 other plugins)

System:
   ios-deploy : 1.9.4
   ios-sim    : 7.0.0
   NodeJS     : v10.14.1 (/usr/local/bin/node)
   npm        : 6.5.0
   OS         : macOS High Sierra
   Xcode      : Xcode 10.1 Build version 10B61
```

**Comment by paulstelzer** [comment]
Thanks! Yes, this is correct. There are other issues here with the same output (e.g. #16863). We are aware of this issue and try to fix it

**Comment by elwinvaneede** [comment]
Ah, I didn't find that one, because it didn't mention the split pane, but I see it's the same behavior. Thanks!

**Comment by elwinvaneede** [comment]
I just checked and this behavior isn't found anymore in @ionic/angular 4.8.0 (and probably for some versions now). Might be fixed by the fix for #16863. I will close this issue. Thanks!
